Warcraft.NET/Files/phys/Entries/PLYTEntry.cs

```diff
@@ -0,0 +1,132 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Warcraft.NET.Files.Structures;
+
+namespace Warcraft.NET.Files.phys.Entries
+{
+    public class PLYTEntry
+    {
+        public PLYT_HEADER header;
+        public PLYT_DATA data;
+
+        [Serializable]
+        public struct PLYT_HEADER
+        {
+            public UInt32 vertexCount;               // Mostly 8
+            public byte[] unk_04;
+            public UInt64 RUNTIME_08_ptr_data_0;  // = &data[i].unk_0
+            public UInt32 count_10;               // Mostly 6
+            public byte[] unk_14;
+            public UInt64 RUNTIME_18_ptr_data_1;  // = &data[i].unk_1
+            public UInt64 RUNTIME_20_ptr_data_2;  // = &data[i].unk_2
+            public UInt32 nodeCount;               // Mostly 24
+            public byte[] unk_2C;
+            public UInt64 RUNTIME_30_ptr_data_3;  // = &data[i].unk_3
+            public float[] unk_38;                 // not sure if floats: has e-08 values
+        }
+
+        [Serializable]
+        public struct PLYT_DATA
```

Comment:
Create a folder Structures inside the Phys folder and move every struct to its own file